lib/src/lints/function_lines_of_code/function_lines_of_code_metric.dart

```diff
@@ -57,6 +60,22 @@ class FunctionLinesOfCodeMetric
     ErrorReporter reporter,
     AstNode node,
   ) {
+    final name = () {
+      if (node is FunctionDeclaration) {
+        return node.name.lexeme;
+      } else if (node is MethodDeclaration) {
+        return node.name.lexeme;
+      } else if (node is FunctionExpression) {
+        return node.declaredElement?.name;
+      } else {
+        return null;
+      }
+    }();
```

Comment:
Please, extract this into a separate private method or just inline it into current method. Do not use inline function declaration since it hurts readability.